mod-data-import-converter-storage-server/src/main/java/org/folio/dao/AbstractProfileDao.java

```diff
@@ -174,6 +174,27 @@
     return promise.future();
   }
 
+  @Override
```

Comment:
We added this checking base on this acceptance criteria, correct?
**if there is a profile that is marked as deleted or not and has the same UUID as UUID from request, then respond as error**
Do you think that profile ids can be the same, because they should be unique by nature? I just asking because this additional call to database can affect performance

Reply:
In most cases, creating job profiles using the UUID will be used when importing data. In this case, it is possible that there may be problems with uniqueness.